package.json

```diff
@@ -36,17 +36,17 @@
     },
     "dependencies": {
         "escape-html": "^1.0.3",
+        "highlight.js": "^11.11.0",
         "just-clone": "^6.2.0",
-        "marked": "^14.1.0",
-        "prismjs": "1.29.0",
+        "marked": "^12.0.2",
```

Comment:
Any specific reason we downgrade the version of marked?

Reply:
Seemed like it was a requirement for highlight.js. I'll double check if we can get it to stay at 14.1.0.

Reply:
Think I managed to bump it up again without issues. Let's see.